_episodes_rmd/04-dplyr.Rmd

```diff
@@ -25,8 +25,10 @@ source("../bin/chunk-options.R")
 knitr_fig_path("04-")
 ```
 
-```{r, echo = FALSE, purl = FALSE, message = FALSE}
-variants <- read.csv("../data/combined_tidy_vcf.csv")
+```{r, echo = FALSE, eval = TRUE, purl = FALSE}
+## silently read in CSV file from FigShare
+
+variants <- read.csv("https://ndownloader.figshare.com/files/14632895")
```

Comment:
What's the rationale for loading the data directly from figshare vs having it on the AMI? More portability of the code, I guess?

Reply:
My immediate rationale is that I needed it in order to make the lesson render, but yes, also portability :-)